dds_web/security/auth.py

```diff
@@ -266,7 +278,8 @@ def __handle_multi_factor_authentication(user, mfa_auth_time_string):
             if mfa_auth_time >= dds_web.utils.current_time() - MFA_EXPIRES_IN:
                 return user
 
-        send_hotp_email(user)
+        if not user.totp_enabled:
+            send_hotp_email(user)
```

Comment:
This returns True or False, so then it continues to check the path below, but if it doesn't pass that if statement then it will say "Please check your primary email" which would be weird if totp is enabled,